packages/slate-editor/src/components/SearchInput/Suggestions.tsx

```diff
@@ -44,42 +46,51 @@ export function Suggestions<T>({
                 name: 'flip',
                 enabled: true,
                 options: {
-                    fallbackPlacements: ['top'],
+                    fallbackPlacements: ['top-end'],
                 },
             } satisfies Partial<FlipModifier>,
-            {
-                name: 'preventOverflow',
-                enabled: true,
-                options: {
-                    altAxis: true,
-                    mainAxis: true,
-                },
-            } satisfies Partial<PreventOverflowModifier>,
         ],
-        placement: 'bottom',
+        placement: 'bottom-end',
     });
 
     const updatePanelSizeAndPosition = useFunction(() => {
         setHeight(childrenContainer.current?.getBoundingClientRect().height);
+
+        if (container.current) {
+            const viewport = document.body.getBoundingClientRect();
+            const rect = container.current.getBoundingClientRect();
+            setMaxHeight(clamp(viewport.height - rect.top - 4, minHeight, maxHeight));
```

Comment:
What is `4` magic number? 🤔

Reply:
I'm not sure but I think it's probably a bottom padding. I restored this code from how it was before.
I can add a constant for it, but I'm gonna rework these dialogues this week during [my project](https://linear.app/prezly/project/refactor-placeholders-in-slate-editor-237d6ef14225) so probably not worth spending extra time on this.